docs/docs-components/data/types.js

```diff
@@ -12,3 +12,64 @@ export type DesignOverview = {|
   path: string,
   svg: SVGElement,
 |};
+
+type StatusType = 'notAvailable' | 'partial' | 'planned' | 'ready';
+
+/**
+ * All components should be in a single list. That list can be filtered by category/platform/etc where needed.
+ */
+
+type ComponentAccessibility = {|
+  a11yComprehension?: StatusType,
+  a11yNavigation?: StatusType,
+  a11yScreenreader?: StatusType,
+  a11yVisual?: StatusType,
+  summary?: StatusType,
+|};
+
+type ComponentCategory =
```

Comment:
Removed the categories we were using for design foundations tiles (see #3004). Those weren't components anyways so were an awkward fit here.